icu4c/source/test/intltest/caltest.cpp

```diff
@@ -4086,4 +4088,69 @@ void CalendarTest::TestChineseCalendarMapping() {
 
 #endif /* #if !UCONFIG_NO_FORMATTING */
 
+void CalendarTest::TestChineseCalendarMonthInSpecialYear(void) {
+    UErrorCode status = U_ZERO_ERROR;
+    GregorianCalendar gc(status);
+    ChineseCalendar cal(Locale::getRoot(), status);
+    if (failure(status, "Constructor failed")) return;
+    struct TestCase {
+      int32_t gyear;
+      int32_t gmonth;
+      int32_t gdate;
+      int32_t cmonth;
+      int32_t cdate;
+      bool cleapmonth;
+    } cases[] = {
+        // Gregorian             Chinese Calendar
+        // First some recent date
+        // From https://www.hko.gov.hk/tc/gts/time/calendar/pdf/files/2022.pdf
+        { 2022, UCAL_DECEMBER, 15, UCAL_NOVEMBER, 22, false},
+        // From https://www.hko.gov.hk/tc/gts/time/calendar/pdf/files/2023.pdf
+        { 2023, UCAL_MARCH, 21, UCAL_FEBRUARY, 30, false},
+        { 2023, UCAL_MARCH, 22, UCAL_FEBRUARY, 1, true},
+        // We know there are some problematic year, especially those involved
+        // the rare cases of M11L and M12L.
+        // Check 1889, 1890, 2033
```

Comment:
You mention 1889 here, but I don't see any tests for 1899 below.

Reply:
done